app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -2442,15 +2444,15 @@ public static String extraStatusLine() {
             if (prefs.getBoolean("status_line_carbs", false)) {
```

Comment:
Changes below improve the look of the extra status line or is there another reason?